src/kOS/Screen/TermWindow.cs

```diff
@@ -14,9 +14,9 @@ namespace kOS.Screen
     public class TermWindow : KOSManagedWindow , ITermWindow
     {
         /// <summary>
-        /// Pixel size of one square section of the font template image file holding one character:
+        /// Pixel size of one square section of the font template image file holding one character.
         /// </summary>
-        private const int CHAR_SOURCE_SIZE = 8;
+        private int charSourceSize;
```

Comment:
While I was at it, I also made it calculate this on the fly from the image file (under the assumption the image file contains exactly 16 character images per row) instead of hardcoding it.  That means we can drop-in replacement font images and see what it looks like, or even let the end-user do that, and they can be whatever size scale we like without recompiling the code.  Thus someone who wants to could make, say, a 192x192 image for font_sml.png, with each character being 12x12 instead of 8x8, and the system would happily use it without recompiling.